### PR TITLE
REM-928: add validation for events params

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**
### Jira references
- Ticket (task) — https://remmeio.atlassian.net/browse/REM-928

### Implemented
— Add validation for batch identifier.

### References
- Web sockets interface — https://remmeio.atlassian.net/wiki/spaces/WikiREMME/pages/437190680/2018-11-14+RFC+1+WebSockets+architecture+and+interface

### CODE REVIEW

File: remme/rpc_api/event/_handlers.py [hunk 1]
```diff
@@ -280,13 +299,24 @@ def parse_evt(self, evt):
...
 
     def validate(self, msg_id, params):
+
         from_block = params.get('from_block')
+        swap_id = params.get('id')
+
         if from_block and not isinstance(from_block, str):
-            raise ClientException(message='Invalid "from_block" type',
-                                  msg_id=id)
+            raise ClientException(message='Invalid "from_block" type', msg_id=id)
```
Comment (DeV1doR) [inline remme/rpc_api/event/_handlers.py]: `from_block` param must be also validated

File: remme/rpc_api/event/_handlers.py [hunk 2]
```diff
@@ -164,12 +165,21 @@ def parse_evt(self, evt):
 
     def validate(self, msg_id, params):
         try:
-            id = params['id']
+            batch_id = params['id']
+
         except KeyError:
             raise RpcInvalidParamsError(message='Missed id', msg_id=msg_id)
 
+        if len(batch_id) != 128:
```
Comment (DeV1doR) [inline remme/rpc_api/event/_handlers.py]: this to check could be united, client do not need to know, what regex and what he does not match, just generalized - `Incorrect id`

File: remme/rpc_api/event/_handlers.py (hunk 1 above)
Comment (dmytrostriletskyi) [reply remme/rpc_api/event/_handlers.py]: @DeV1doR, it is not scope of the ticket. @eugene-babichenko, add validation for `from_block` also in this pull request?

File: remme/rpc_api/event/_handlers.py (hunk 2 above)
Comment (dmytrostriletskyi) [reply remme/rpc_api/event/_handlers.py]: @DeV1doR, done.

File: remme/rpc_api/event/_handlers.py (hunk 1 above)
Comment (dmytrostriletskyi) [reply remme/rpc_api/event/_handlers.py]: @eugene-babichenko, done.

**Pull request merged**